### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Meri-MG**
The Awesome book's application was modified to have:

A Navigation bar.
Three content sections:
Books list.
Add book form.
Contact info.

P.S. There is going a new webhint linter error which origin nobody knows yet in our module and in the previous module Micronauts are facing the same ---Run npx hint --telemetry=off .
Invalid option '--telemetry' - perhaps you meant '--help'?
Error: Process completed with exit code 1.  --- 
So, if it's still possible to review this code, Please do so.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
     <title>Awesome Books</title>
 </head>
+
 <body>
+    <nav class="navContainer">
+        <h1 class="title1"><span>Awesome </span> Books</h1>
+        <img src="images/nav.svg" alt="navicon" class="navImg d-none">
+        <img src="images/closeNav.svg" alt="closeNav" class="closeImg d-none">
+        <ul class="navUl">
+            <li id="bookList" class="navLi pointer">Book list</li>
```
Comment (rloterh) [inline index.html:30]: - [ ] OPTIONAL
It is good practice to use **all lowercase with hyphens** separating words, for your CSS naming convertion

File: index.html [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+            <li id="addNew" class="navLi pointer">Add New</li>
+            <li id="contactInfo" class="navLi pointer">Contact info</li>
+        </ul>
+    </nav>
```
Comment (rloterh) [inline index.html:34]: - [ ] To enhance the user experience of this application kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your nav menu:**
image


**The wireframe nav menu:**
image

File: index.html [hunk 3]
```diff
@@ -21,10 +54,12 @@ <h2>Add a New <span>Book</span></h2>
...
         <label for="author">
-            <input class="input-author" id="author" type="text" maxlength="15" minlength="3" placeholder="Author" required>
+            <input class="input-author" id="author" type="text" maxlength="15" minlength="3" placeholder="Author"
+                required>
         </label>
         <button type="submit" class="add-btn">Add</button>
     </form>
     <script src="./js/classes.js"></script>
 </body>
+
```
Comment (rloterh) [inline index.html:64]: The **footer section is missing** from your pages.
- [ ] Kindly add a _footer section_ to your pages, which contains the text `Copyright`, and is aligned to the left of the _footer_.
- [ ] The _footer_ should **stick to the bottom** of the page.

Your sample Add New page:
image

**The wireframe pages:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Meri & Didi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Correctly used the Luxon library
- Design mostly matches the wireframe
- App is running as intended
- Good ReadMe
- Nice UI design
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**